source/tool/chipsec/hal/spi_uefi.py

```diff
@@ -454,8 +454,12 @@ def traverse_uefi_region( _uefi, data, fwtype, uefi_path='', printall=True, dump
                             if findall: found = True
                             else: return True
                     elif fwbin.Type == EFI_FV_FILETYPE_RAW:
-                        if fwbin.Name == NVAR_NVRAM_FS_FILE and fwbin.UD:
-                            if dumpall: _uefi.parse_EFI_variables( os.path.join(fwbin_dir, 'DEFAULT_NVRAM'), FvImage, False, FWType.EFI_FW_TYPE_NVAR )
+                        if fwbin.Name == NVAR_NVRAM_FS_FILE: 
+                            if dumpall:
+                                if fwbin.UD:
+                                    _uefi.parse_EFI_variables( os.path.join(fwbin_dir, 'NVRAM.UD'), fv.Image, False, FWType.EFI_FW_TYPE_NVAR )
+                                else:
+                                    _uefi.parse_EFI_variables( os.path.join(fwbin_dir, 'NVRAM'), fv.Image, False, FWType.EFI_FW_TYPE_NVAR )

```

Comment:
Suggest to shorten to:
_uefi.parse_EFI_variables( os.path.join(fwbin_dir, 'NVRAM%s' % ('.UD' if fwbin.UD else '')), FvImage, False, FWType.EFI_FW_TYPE_NVAR )